doc/contributor/project-layout.md

```diff
@@ -0,0 +1,33 @@
+# Project Layout
+
+Each folder in the TruffleRuby project is described below:
+
+## `/bench/`
+Files for running benchmarks. See the `jt benchmark` command.
+
+## `/bin/`
+Scripts used to for common Ruby commands. E.g. `ruby`, `gem`.
+
+## `/doc/`
+Project documentation for users and contributors.
+
+## `/lib/`
+Contains the stdlib, core, and TruffleRuby specific Ruby and C libraries.
+
+## `/logo/`
+TruffleRuby logo assets.
+
+## `/mx.truffleruby/`
+Files used by the `mx` build system.
+
+## `/spec/`
+Ruby Spec suite, tags, runner and additional TruffleRuby specific specs.
+
+## `/src/`
+Sources and tests implementing the Ruby lanaguage, core, C extension API, and stdlib.
```

Comment:
`language`.
No stdlib in `src/`.

Reply:
There is only a couple Java tests under `src/`, I would skip in this description or precise it's Java tests.

Reply:
Fixed these, thanks.